apps/frontend/src/libs/components/header/libs/components/notifications-popover/libs/constants/default-page-size.constant.ts

```diff
@@ -0,0 +1,3 @@
+const PAGE_SIZE = 10;
```

Comment:
and file name should be the same
```suggestion
const NOTIFICATIONS_PAGE_SIZE = 10;
```